src/components/teacher/teacher-dashboard.tsx

```diff
@@ -37,7 +36,7 @@ export class TeacherDashboardComponent extends BaseComponent<IProps, IState> {
   };
 
   public render() {
-    const {supports, unit} = this.stores;
+    const {appConfig, supports} = this.stores;
     const {activeTab} = this.state;
 
```

Comment:
We can likely do away with the additional newline?
```suggestion
```

Reply:
Could, but I'm not inclined to make extraneous formatting changes that aren't directly related to the changes in the PR (which is already big enough).